python-backend/tests/status/resources/test_mine_status_resource.py

```diff
@@ -1,19 +1,17 @@
 import json, uuid
 
-from app.api.constants import MINE_STATUS_OPTIONS
-
 
 # GET
 def test_get_mine_status_option(test_client, db_session, auth_headers):
     get_resp = test_client.get('/mines/status', headers=auth_headers['full_auth_header'])
     get_data = json.loads(get_resp.data.decode())
-    assert get_data == {'options': MINE_STATUS_OPTIONS}
+    assert len(get_data.get('records')) == 14
     assert get_resp.status_code == 200
 
 
 def test_get_mine_status_not_found(test_client, db_session, auth_headers):
     get_resp = test_client.get(
         '/mines/status/' + str(uuid.uuid4()), headers=auth_headers['full_auth_header'])
     get_data = json.loads(get_resp.data.decode())
-    assert get_data['error']['message'] == 'Mine Status not found'
+    assert get_data['message'] == '404 Not Found: Mine Status not found'
```

Comment:
I think we've moved toward a case-insensitive substring match on errors now, making it modestly useful but more flexible. Not saying either one is right or wrong, but I think that's how our newer tests have been written

Reply:
I haven't heard or seen anything on that, but I changed this to just null check the message instead.